pkg/login/social/github_oauth.go

```diff
@@ -153,27 +153,33 @@ func (s *SocialGithub) HasMoreRecords(headers http.Header) (string, bool) {
 }
 
 func (s *SocialGithub) FetchOrganizations(client *http.Client, organizationsUrl string) ([]string, error) {
+	url := organizationsUrl
```

Comment:
Could you please add the `?per_page=100` query parameter to make this coherent with the request url in the`FetchTeamMemberships` function?

Reply:
Adjusted